docs/USAGE-GUIDE.md

```diff
@@ -1585,6 +1585,103 @@ int s2n_config_add_ticket_crypto_key(struct s2n_config *config, const uint8_t *n
 **s2n_config_add_ticket_crypto_key** adds session ticket key on the server side. It would be ideal to add new keys after every (encrypt_decrypt_key_lifetime_in_nanos/2) nanos because
 this will allow for gradual and linear transition of a key from encrypt-decrypt state to decrypt-only state.
 
+## TLS1.3 Pre-Shared Key (PSK) Related Calls 
+
+S2N-TLS provides support for pre-shared keys that can be established out of band and pre-shared keys that can be established in a previous connection and then used to establish a new connection via session resumption for versions >= TLS1.3. The Pre-shared key is used to mutually authenticate the peers, under the assumption that it is not shared with others. This allows the parties to skip the certificate verification steps, saving bandwidth, communication rounds, and latency. When a PSK is used, the underlying assumption is that a party receiving a message that passes verification with the PSK knows that the message was sent by a party that also owns that PSK. 
+
+Benefits of using Pre-Shared Keys include: 
+
+- Using pre-shared keys can avoid the need for public key operations. This is useful if TLS is used in  performance-constrained environments with limited CPU power.
+- Pre-shared keys may be more convenient from a key management point of view. For instance, in closed environments where the connections are  mostly configured manually in advance, it is easier to configure a PSK  than to use certificates. Another case is when the parties already have a mechanism for setting up a shared secret key, and that mechanism could be used to "bootstrap" a key for authenticating a TLS connection.
+- Pre-shared keys are more appropriate for use cases with long-lived credentials stored in hardware secure elements than PKI-based solutions which enforce expiration dates and require error-prone rotation schemes.
```

Comment:
Can you tweak the phrasing of your bullets? The intro to this list says "benefits include:", so bullets should follow from that. Trying reading it as an in-line list: "Benefits of using pre-shared keys include using pre-shared keys can avoid the need for public key operations, pre-shared keys may be more convenient from a key management point of view, etc."

Also: nit, but you've got something going on with your spaces? Sometimes there's two spaces between words instead of one